lib/resty/limit/conn.lua

```diff
@@ -23,6 +23,29 @@ local mt = {
 }
 
 
+local function incr_or_init(key, dict)
```

Comment:
I think use the shdict native API is better. There is a pending PR for this:

https://github.com/openresty/lua-nginx-module/pull/579